application/values-test.yaml

```diff
@@ -135,7 +135,16 @@ deployment:
     tag: v1.0.0
     pullPolicy: IfNotPresent
 
-  # Readiness and Liveness probes
+  # Startup, Readiness and Liveness probes
+  startupProbe:
+    enabled: false
```

Comment:
should be true here. as these values are for validation in pipeline

Reply:
fixed